src/qunit.cpp

```diff
@@ -1000,7 +1072,7 @@ void QUnit::CNOT(bitLenInt control, bitLenInt target)
             CNOT(target, control);
         } else if (norm(tShard.amp0) < min_norm) {
             std::swap(cShard.amp0, cShard.amp1);
-            if (cShard.unit->GetQubitCount() == 1) {
+            if (cShard.unit->GetQubitCount() == 1U) {
```

Comment:
Don't you have a ONE macro for this?

Reply:
We have a `ONE_BCI` macro, i.e. 1 for `bitCapInt`, but this is a `bitLenInt`. The unsigned default 1 is correct, in this case, whether it should have its own macro.